src/core/Global.ttslua

```diff
@@ -302,12 +302,34 @@ function drawChaosToken(params)
 
     -- add the token to the list, compute new position based on list length
     tokenOffset[1] = tokenOffset[1] + (0.17 * #chaosTokens)
-    local token = chaosBag.takeObject({
-      index = 0,
-      position = params.mat.positionToWorld(tokenOffset),
-      rotation = params.mat.getRotation()
-    })
+        
```

Comment:
There should be `local token` at this point (otherwise the code below will set the global `token`)